packages/cli/templates/web-library-package/package.json.hbs

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "{{name}}",
+  "version": "{{pluginVersion}}",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+{{#if privatePackage}}
+  "private": {{privatePackage}},
+{{/if}}
+  "publishConfig": {
+{{#if npmRegistry}}
+  "registry": "{{npmRegistry}}",
+{{/if}}
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "backstage": {
+    "role": "web-library"
+  },
+  "scripts": {
+    "start": "backstage-cli package start",
+    "build": "backstage-cli package build",
+    "lint": "backstage-cli package lint",
+    "test": "backstage-cli package test",
+    "clean": "backstage-cli package clean",
+    "prepack": "backstage-cli package prepack",
+    "postpack": "backstage-cli package postpack"
+  },
+  "peerDependencies": {
+    "react": "{{versionQuery 'react' '^16.13.1 || ^17.0.0'}}"
```

Comment:
@jhaals should we remove react peerDependency or leave it?

Reply:
In general I don't think we should have dependencies that are not used but this is a template and I guess you'll end up needing these in the majority of cases. @Rugvip WDYT?

Reply:
I discussed this with Rugvip and concluded that we should removed all dependencies that aren't used to keep the dependency section as minimal as possible.